internal/command/command.go

```diff
@@ -131,6 +130,11 @@ func (c Command) AddToParent(parent *cobra.Command) {
 
 		handleError("Command Error", err)
 
+		resultString := getFullCommand(c.Cmd)
+		err = util.SendEvent(resultString)
+		if err != nil {
+			fmt.Printf(err.Error())
```

Comment:
I think here we should only report the problem to Sentry and not output it at all.

Reply:
addressed, will be implemented in the next pr that uses this pr's commit for flowkit